feature-toggle-api/templates/cronjob.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "hmcts.releaseName" . }}
```

Comment:
This would be better named something like: {{ template "hmcts.releaseName" . }}-importer

I also agree that naming this file cronjob.yaml is not ideal as it doesn't say what it's for.

Reply:
agreed - will change this